tests/integration/test_analysisd/test_eps/test_process_old_events_instead_new_events.py

```diff
@@ -0,0 +1,206 @@
+        - set_wazuh_configuration_eps:
+            type: fixture
+            brief: Set the wazuh configuration according to the configuration data.
+        - configure_internal_options_eps:
+            type: fixture
+            brief: Set the wazuh internal option configuration according to the configuration data.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - delete_alerts_folder:
+            type: fixture
+            brief: Delete all the content od the /var/log/alerts folder.
+        - restart_wazuh_daemon_function:
+            type: fixture
+            brief: Restart all the wazuh daemons.
+        - simulate_agent:
+            type: fixture
+            brief: Execute a script that simulate agent and send `logcolector` logs to the manager.
+
+    assertions:
+        - The timestamp of the oldest numerated messages have to be lower than he new messages.
+
+    input_description:
+        - The `cases_process_old_events_one_thread.yaml` file provides the module configuration for this test.
+    '''
+    # Set logcollector message that the agent sents
+    logcollector_message = 'Invalid user random_user from 172.17.1.1 port 56550:Message number:'
+    # Set the alerts start message
+    start_alert_msg = '** Alert '
+    # Initial timestamp to compare
+    timestamp_bkp = datetime.fromtimestamp(float(0.0)).strftime('%Y-%m-%d %H:%M:%S')
+    # Factor to iterate the alerts.log file to reduce the test execution time
+    time_events_processed = 5
+
+    # Wait 'timeframe' / 2 second to read the wazuh-analysisd.state to ensure that has corrects values
+    sleep(metadata['timeframe'] / 2)
+    events_processed = int(evm.get_analysisd_state('events_processed'))
+    events_received = int(evm.get_analysisd_state('events_received'))
+
+    # Check that the timestamp of the message in the alerts.log is lower than the next one
+    # In order to reduce the test time execution, It will check {time_events_processed} consecutive timeframe
+    # by checking events_processed * time_events_processed
+    if(events_processed * time_events_processed <= events_received):
+        for index in range((events_processed * time_events_processed) - 1):
+            # Get the timestamp of the log
+            timestamp = evm.get_alert_timestamp(start_alert_msg, f"{logcollector_message} {index}")
+            # Check that the timestamp of the first message y lower than the previous one
+            assert timestamp >= timestamp_bkp, 'The timestamp of the previous message has to be lower than the '\
+                                               'next one'
+            # Store the timestamp to be compared with the next one
+            timestamp_bkp = timestamp
+    else:
+        raise Exception('Not enough messages were sent. Please increase the `total_msg` for this test.')
```

Comment:
This message is confusing, why does the user need to change the test?

Reply:
Ok

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3181/commits/f817116435639fa720611bd68eb90ced7c606062)